tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -135,16 +135,16 @@ def test_inaccessible_project(self):
         assert response.data == {"detail": "You do not have permission to perform this action."}
 
     def test_unknown_field(self):
-        response = self.do_request({"field": ["summ(sessin)"]})
+        response = self.do_request({"field": ["summ(session)"]})
 
         assert response.status_code == 400, response.content
-        assert response.data == {"detail": 'Invalid field: "summ(sessin)"'}
+        assert response.data == {"detail": 'Invalid field: "summ(session)"'}
 
     def test_unknown_groupby(self):
-        response = self.do_request({"field": ["sum(session)"], "groupBy": ["envriomnent"]})
+        response = self.do_request({"field": ["sum(session)"], "groupBy": ["environment"]})
 
         assert response.status_code == 400, response.content
-        assert response.data == {"detail": 'Invalid groupBy: "envriomnent"'}
+        assert response.data == {"detail": 'Invalid groupBy: "environment"'}
```

Comment:
```suggestion
        assert response.data == {"detail": 'Invalid groupBy: "_environment"'}
```

Reply:
Looks like this was resolved in 8b13ba0e0 (sorry, these splits aren't quite as systematic as I've done for other projects which were strict splittable directories....)